package.json

```diff
@@ -73,7 +73,7 @@
     "zone.js": "^0.6.17"
   },
   "peerDependencies": {
-    "@angular/core": "2.0.0-rc.3 || ^2.0.0-rc.4 || ^2.0.0-rc.5 || ^2.0.0-rc.6",
```

Comment:
@e-schultz since we're in beta for ng2-redux v4, I took the opportunity to clean this up as well. Does that make sense to you?